File: solutions/beecrowd/1367/1367.cpp
#include <cstdint>
#include <cstdio>
#include <cstring>
#include <iostream>
#include <string>

int main() {
    std::int16_t n, time, total_time, count, extra_time[26];
    char letter;
    std::string status, letters = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";

    while (scanf("%d", &n) && n) {
        total_time = 0;
        count = 0;
        memset(extra_time, 0, sizeof(extra_time));

        while (n--) {
            std::cin >> letter >> time >> status;

            if (status == "correct") {
                total_time += time;
                total_time += extra_time[letters.find(letter)];
                count++;
            } else {
                extra_time[letters.find(letter)] += 20;
            }
        }

        printf("%d %d\n", count, total_time);
    }

    return 0;
}

File: solutions/beecrowd/1367/1367.py
import sys
from string import ascii_uppercase

input = sys.stdin.readline
output = sys.stdout.write

while line := input().strip():
    if line == '0':
        break

    n = int(line)

    total_time = 0
    count = 0
    extra_time = [0] * 26

    for _ in range(n):
        letter, time, status = input().split()
        time = int(time)

        if status == 'correct':
            total_time += time
            total_time += extra_time[ascii_uppercase.index(letter)]
            count += 1
        else:
            extra_time[ascii_uppercase.index(letter)] += 20

    output(f'{count} {total_time}\n')
